lms/templates/admin/application_instance/show.html.jinja2

```diff
@@ -12,12 +12,16 @@
         {% endif %}
     </span>
 {% endblock %}
-{% macro settings_checkbox(label, setting, sub_setting, field_name, default=False) %}
+{# Tri-state checkbox with enabled/disabled/use-default options. #}
```

Comment:
It looks to me like a tri-state is appropriate for all checkboxes we currently have on this page. We might have a need for a regular non-tri-state checkbox in future, in which case we'd rename this `settings_tristate_checkbox` or `settings_flag_checkbox`.